launch/aruco_hand_eye.launch

```diff
@@ -1,19 +1,21 @@
 <launch>
   <!-- Set the eye mode (see README for more info) -->
-  <arg name="camera_parent_frame"/>
-  <arg name="marker_parent_frame"/>
+  <arg name="camera_parent_frame" default="/ee_link"/>
```

Comment:
at least some of the changes in this file seem particular to your setup, I think they'd need to be removed before merging. Is that right?